app/views/application/_working_tasks_table.erb

```diff
@@ -0,0 +1,14 @@
+<div class='tasklist'>
+  <table>
+    <% @working_tasks.each do |task|%>
+      <tr>
+         <td id='task'><%= link_to task.name, "/tasks/#{task.id}" %></td>
```

Comment:
Same comment re: using IDs inside of an each loop, definitely recommend switching these to classes